lib/vanagon/platform/windows.rb

```diff
@@ -37,6 +37,25 @@ def package_name(project)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
+        # HOW DO I WORK OUT the Package Type Here - Can I use the binding as per next statement ??
+        # Or should we be splitting out this into a nuget and a windows sub-class - i.e. too many package_type
+        package_type = binding.eval("@project.platform.package_type")
```

Comment:
@melissa Is this the best way to pickup the package type - is it even legal Ruby ?

Think we need to sub-class the windows.rb file into two separate msi and nuget sub-classes as we appear to be using this conditional rather too many times.